Parse/src/main/java/com/parse/ManifestInfo.java

```diff
@@ -462,10 +461,6 @@ private static boolean hasAnyGcmSpecificDeclaration() {
     return false;
   }
 
-  private static boolean isGooglePlayServicesAvailable() {
-    return Build.VERSION.SDK_INT >= 8 && getPackageInfo("com.google.android.gsf") != null;
```

Comment:
Hah, this reminds me that this code was originally written to work on Donut phones.